### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Deepak-du**
Readme file has been updated in a detailed way.

### CODE REVIEW

**Comment by Deepak-du** [comment]
can you review this please @orta

File: README.md [hunk 1]
```diff
@@ -1,7 +1,18 @@
 # Source Repository for graphql.org
 
-This repository contains the source code of https://graphql.org website.
+This repository contains the source code of [GrapQL](https://graphql.org)
```
Comment (mikeesto) [inline README.md:3]: Small typo:

```suggestion
This repository contains the source code of [GraphQL](https://graphql.org)
```

Comment (carolstran) [reply README.md:3]: I'd also add that this is the source code for the GraphQL _website_ (as to not be confused with the spec)

```suggestion
This repository contains the source code for the [GraphQL website](https://graphql.org).
```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
+> [Invite link to slack channel](https://graphql-slack.herokuapp.com/)
```
Comment (carolstran) [inline README.md:4]: Maybe we can mention that there's a specific Slack channel dedicated to the website? 

Perhaps something like...

```suggestion
> You can find more discussions on the [#website channel on the GraphQL Slack](https://graphql.slack.com/messages/website/). [Get your invite here!](https://graphql-slack.herokuapp.com/) 
```

File: README.md [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
 
-## Contributing
+**GraphQL 🎉** is a query language for APIs and a runtime for fulfilling those queries with your existing data.
 
-Check out our [contributing guide](./CONTRIBUTING.md) for detailed instructions for how to make changes to graphql.org 🎉
\ No newline at end of file
+**GraphQL 🎉** provides a complete and understandable description of the data in your API, gives clients the power to ask for exactly what they need and nothing more, makes it easier to evolve APIs over time, and enables powerful developer tools.
```
Comment (carolstran) [inline README.md]: Curious, why have two descriptions instead of merging them into one?

File: README.md [hunk 4]
```diff
@@ -1,7 +1,18 @@
...
+
+## Documentation
+- [Site](https://graphql.org/)
+- [User guide](https://graphql.org/code/)
+- [API documentation](https://graphql.org/learn/)
+
+## How to contribute
+If you want to report a bug or if you just want to request for a new feature/improvement Check out our [contributing guide](./CONTRIBUTING.md) for detailed instructions for how to make changes to [Graphql 🎉](https://graphql.org/)
+
+Thanks for taking time to contribute to Graphql 🎉 👍
```
Comment (carolstran) [inline README.md:18]: I'm maybe biased but I'd keep the original copy as it's succinct and there are many more ways for people to contribute to the docs 😊

File: README.md [hunk 5]
```diff
@@ -1,7 +1,18 @@
...
-## Contributing
+**GraphQL 🎉** is a query language for APIs and a runtime for fulfilling those queries with your existing data.
 
-Check out our [contributing guide](./CONTRIBUTING.md) for detailed instructions for how to make changes to graphql.org 🎉
\ No newline at end of file
+**GraphQL 🎉** provides a complete and understandable description of the data in your API, gives clients the power to ask for exactly what they need and nothing more, makes it easier to evolve APIs over time, and enables powerful developer tools.
+
+## Documentation
+- [Site](https://graphql.org/)
+- [User guide](https://graphql.org/code/)
```
Comment (carolstran) [inline README.md:12]: The `Code` page isn't exactly a user guide, it's more of a list of available libraries, tools, and services. 

I'd suggest updating it to something like this:
```suggestion
- [Language support, tools, and services](https://graphql.org/code/)
```

File: README.md [hunk 6]
```diff
@@ -1,7 +1,18 @@
...
+- [API documentation](https://graphql.org/learn/)
```
Comment (carolstran) [inline README.md:13]: `API` isn't quite right here, as there's a separate site for the [spec documentation](https://spec.graphql.org/) (which could also be worth noting in this section). `Reference` might be a more appropriate word choice.

```suggestion
- [Reference documentation](https://graphql.org/learn/)
```

I'd also move this point above the Code page as it's the most prominent part of the site.

File: README.md [hunk 7]
```diff
@@ -1,7 +1,18 @@
...
+
```
Comment (carolstran) [inline README.md:14]: I'd opt for featuring all of the main pages in this section so people are able to get an overview of what is available on the site.

```suggestion
- [Frequently Asked Questions (FAQ)](https://graphql.org/faq/)
- [Community resources](https://graphql.org/community/)

```

**Review by carolstran — CHANGES_REQUESTED** [review]
Thank you so much for taking the time to update the README 🙌🏼  🎉  It's great to give more context to what this repo contains and what's available through graphql.org. 

Overall, I like the structure - but added some suggestions as far as word choice 😊

File: README.md [hunk 8]
```diff
@@ -1,7 +1,20 @@
 # Source Repository for graphql.org
 
-This repository contains the source code of https://graphql.org website.
+This repository contains the source code of [GraphQL](https://graphql.org)
+> You can find more discussions on the [#website channel on the GraphQL Slack](https://graphql.slack.com/messages/website/). [Get your invite here!](https://graphql-slack.herokuapp.com/)
```
Comment (carolstran) [inline README.md:4]:
```suggestion
This repository contains the source code for the [GraphQL website](https://graphql.org).

> You can find more discussions on the [#website channel on the GraphQL Slack](https://graphql.slack.com/messages/website/). [Get your invite here!](https://graphql-slack.herokuapp.com/) 
```

File: README.md [hunk 9]
```diff
@@ -1,7 +1,20 @@
...
+**GraphQL 🎉** is a query language for APIs and a runtime for fulfilling those queries with your existing data.
 
-Check out our [contributing guide](./CONTRIBUTING.md) for detailed instructions for how to make changes to graphql.org 🎉
\ No newline at end of file
+**GraphQL 🎉** provides a complete and understandable description of the data in your API, gives clients the power to ask for exactly what they need and nothing more, makes it easier to evolve APIs over time, and enables powerful developer tools.
+
+## Documentation
+- [Site](https://graphql.org/)
+- [Language support, tools, and services](https://graphql.org/code/)
+- [Reference documentation](https://graphql.org/learn/)
```
Comment (carolstran) [inline README.md:13]:
```suggestion
- [Reference documentation](https://graphql.org/learn/)
- [Language support, tools, and services](https://graphql.org/code/)
```

File: README.md (hunk 4 above)
Comment (carolstran) [reply README.md:20]: So here's my suggestion that would combine the old language and the new reference to the GraphQL website:

```suggestion
Check out our [contributing guide](./CONTRIBUTING.md) for detailed instructions on how to make changes to the GraphQL website 🎉
```

File: README.md [hunk 10]
```diff
@@ -1,7 +1,20 @@
 # Source Repository for graphql.org
 
-This repository contains the source code of https://graphql.org website.
+This repository contains the source code of [GraphQL](https://graphql.org)
+> You can find more discussions on the [#website channel on the GraphQL Slack](https://graphql.slack.com/messages/website/). [Get your invite here!](https://graphql-slack.herokuapp.com/) 
 
-## Contributing
+**GraphQL 🎉** is a query language for APIs and a runtime for fulfilling those queries with your existing data.
```
Comment (carolstran) [inline README.md:6]: Removing this and adding to the next paragraph:

```suggestion
```

File: README.md [hunk 11]
```diff
@@ -1,7 +1,20 @@
...
 
-Check out our [contributing guide](./CONTRIBUTING.md) for detailed instructions for how to make changes to graphql.org 🎉
\ No newline at end of file
+**GraphQL 🎉** provides a complete and understandable description of the data in your API, gives clients the power to ask for exactly what they need and nothing more, makes it easier to evolve APIs over time, and enables powerful developer tools.
```
Comment (carolstran) [inline README.md:8]: Combining the two descriptions, plus adding a note about the spec:

```suggestion
A brief overview: **GraphQL** is a query language for APIs and a runtime for fulfilling those queries with your existing data. It provides a complete and understandable description of the data in your API, gives clients the power to ask for exactly what they need, and enables powerful developer tools. The [specification](https://spec.graphql.org/) is open source and governed by the [GraphQL Foundation](https://foundation.graphql.org/). 
```

**Review by carolstran — CHANGES_REQUESTED** [review]
Put all of my other suggestions into proper commit suggestions as that might be easier to work with. If you disagree or want to discuss any, please say so! I'm not super attached to any of these changes so happy to discuss 😊

**Comment by Deepak-du** [comment]
Thanks for correcting me @carolstran😊🚀

**Review by carolstran — APPROVED** [review]
Amazing! Thank you 🎊

**Pull request merged**